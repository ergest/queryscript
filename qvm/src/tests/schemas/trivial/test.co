let __test_output = 1;
