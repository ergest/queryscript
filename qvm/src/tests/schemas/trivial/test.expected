{
    "__test_output": Int64(
        1,
    ),
    "types": {
        "let __test_output": Int64,
    },
}